blockchain_publisher/src/main/java/org/cardanofoundation/lob/app/blockchain_publisher/service/event_publish/LedgerUpdatedEventPublisher.java

```diff
@@ -49,8 +49,7 @@ public void sendTxLedgerUpdatedEvents(String organisationId,
                         val publishStatusM = txEntity.getL1SubmissionData().flatMap(L1SubmissionData::getPublishStatus);
                         val cardanoFinalityScoreM = txEntity.getL1SubmissionData().flatMap(L1SubmissionData::getFinalityScore);
                         val ledgerDispatchStatus = blockchainPublishStatusMapper.convert(publishStatusM, cardanoFinalityScoreM);
-
-                        return new TxStatusUpdate(txEntity.getId(), ledgerDispatchStatus);
+                        return new TxStatusUpdate(txEntity.getId(), ledgerDispatchStatus,txEntity.getL1SubmissionData().flatMap(L1SubmissionData::getTransactionHash));
```

Comment:
Here you put list of blockchain receipts but it will always be list of one receipt for now ("CARDANO_L1") as type.